src/routes/living-manage-offer/components/ExtendOfferManager.js

```diff
@@ -45,7 +45,7 @@ export class ExtendOfferManager extends React.Component {
         </SelectInput>
         <FormControl>
           <SubmitButton validateAll={this.validateAll} type='submit' disabled={this.props.sending}>
-            {this.props.sending ? 'Wird verlängert...' : 'Mietangebot Verlängern'}
+            {this.props.sending ? 'Wird erneuert...' : 'Mietangebot Erneuern'}
```

Comment:
`Mitangebot erneuern`

Reply:
the button always displays text uppercase but you're right 👍